### PR TITLE
Item image extract

### LINKED ISSUE #45: Fancy feed item cards
Collapsible "Read More" for description.

Render HTML in description as image, link, etc.

### PR DESCRIPTION
**Pull request by swglad**
Resolves #45

### CODE REVIEW

**Comment by swglad** [comment]
BUG: voting buttons force reveal to disappear. May want to move them.

File: sourcemash/frontend/static/js/views/item_view.js [hunk 1]
```diff
@@ -11,17 +12,19 @@ Sourcemash.Views.ItemView = Backbone.View.extend({
...
-	upvote: function() {
+	upvote: function(e) {
+		e.stopPropagation()
 		this.model.save({'vote': 1, 'voteSum': this._getNewVoteSum(1)},
 						{success: this.voted});
 	},
 
-	downvote: function() {
+	downvote: function(e) {
+		e.stopPropagation()
```
Comment (alexgerstein) [inline sourcemash/frontend/static/js/views/item_view.js]: do we need stoppropagation anymore?

File: sourcemash/frontend/static/js/views/item_view.js [hunk 2]
```diff
@@ -11,17 +12,19 @@ Sourcemash.Views.ItemView = Backbone.View.extend({
 	  	'click .downvote': 'downvote'
 	},
 
-	upvote: function() {
+	upvote: function(e) {
+		e.stopPropagation()
```
Comment (alexgerstein) [inline sourcemash/frontend/static/js/views/item_view.js]: do we need stoppropagation anymore?

If not, then we don't need e either.

File: sourcemash/frontend/static/js/views/item_view.js (hunk 1 above)
Comment (swglad) [inline sourcemash/frontend/static/js/views/item_view.js]: No, feel free to delete it (based on how many comments you have)

**Pull request merged**